### PR TITLE
implementing docker setup to run opa locally

### PR DESCRIPTION
**Pull request by andy2great**
WATCH OUT! 

OPA is entering the repo
image

### CODE REVIEW

**Comment by andy2great** [comment]
theres a bug where it does not update build file on docker-compose up, need to investigate bit more

File: opa/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM openpolicyagent/opa:latest-debug
+USER root
+VOLUME  ./build
```
Comment (samuel-poirier) [inline opa/Dockerfile]: There is an extra space

File: opa/build/bundle.tar.gz
Comment (samuel-poirier) [inline opa/build/bundle.tar.gz:1]: Should be got ignored

File: opa/Dockerfile (hunk 1 above)
Comment (andy2great) [reply opa/Dockerfile]: you got great eyes 👀

**Review by samuel-poirier — APPROVED** [review]

**Pull request merged**